### ISSUE TITLE
error when write_grads=T in tensorboard callback

### ISSUE THREAD

**Issue by philippmuench** [hiddengenome/deepG#37]
```R
Error in py_call_impl(callable, dots$args, dots$keywords) :
  InvalidArgumentError: You must feed a value for placeholder tensor 'activation_target' with dtype float and shape [?,?]
         [[{{node activation_target}} = Placeholder[dtype=DT_FLOAT, shape=[?,?], _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]
         [[{{node cu_dnnlstm/recurrent_kernel_0_grad/values/_883}} = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_3934_cu_dnnlstm/recurrent_kernel_0_grad/values", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

Detailed traceback:
  File "/home/pmuench/anaconda3/envs/r-reticulate/lib/python2.7/site-packages/tensorflow/python/keras/engine/training.py", line 2177, in fit_generator
    initial_epoch=initial_epoch)
  File "/home/pmuench/anaconda3/envs/r-reticulate/lib/python2.7/site-packages/tensorflow/python/keras/engine/training_generator.py", line 197
```

seems this is a known bug here: https://github.com/rstudio/tensorflow/issues/359

**Comment by philippmuench** [comment]
seems this is a known bug here: https://github.com/rstudio/tensorflow/issues/359

**Comment by philippmuench** [comment]
fixed by using TF 2.0.0
